scipy/stats/_qmc.py

```diff
@@ -976,32 +1005,60 @@ class LatinHypercube(QMCEngine):
     >>> qmc.discrepancy(sample)
     0.0176...  # random
 
+    Use the `strength` keyword argument to produce an orthogonal array based
+    LHS of strength 2. The construction imposes to use a square of a prime
+    number.
+
+    >>> sampler = qmc.LatinHypercube(d=2, strength=2)
+    >>> sample = sampler.random(n=9)
+    >>> qmc.discrepancy(sample)
+    0.00526...  # random
+
+    Options could be combined as a same time to produce an optimized centered
```

Comment:
```suggestion
    Options could be combined to produce an optimized centered
```
The phrase would be "at the same time", but this is implied by "combined", so it's not needed.